hex/sd_api_v3/sdk122_connectivity.patch

```diff
@@ -15,17 +24,24 @@ diff --git a/components/serialization/common/ser_config.h b/components/serializa
 index 1b31fcb..9cc334e 100644
 --- a/components/serialization/common/ser_config.h
 +++ b/components/serialization/common/ser_config.h
-@@ -70,8 +70,8 @@ extern "C" {
+@@ -40,6 +40,6 @@ 
+ /** Max packets size in serialization HAL Transport layer (packets before adding PHY header i.e.
+  *  packet length). */
+-#define SER_HAL_TRANSPORT_APP_TO_CONN_MAX_PKT_SIZE    (uint32_t)(384)
+-#define SER_HAL_TRANSPORT_CONN_TO_APP_MAX_PKT_SIZE    (uint32_t)(384)
++#define SER_HAL_TRANSPORT_APP_TO_CONN_MAX_PKT_SIZE    (uint32_t)(500)
++#define SER_HAL_TRANSPORT_CONN_TO_APP_MAX_PKT_SIZE    (uint32_t)(500)
+
+ #define SER_HAL_TRANSPORT_MAX_PKT_SIZE ((SER_HAL_TRANSPORT_APP_TO_CONN_MAX_PKT_SIZE) >= \
+@@ -70,7 +70,7 @@ extern "C" {
  
  /** UART transmission parameters */
  #define SER_PHY_UART_FLOW_CTRL          APP_UART_FLOW_CONTROL_ENABLED
 -#define SER_PHY_UART_PARITY             true
--#define SER_PHY_UART_BAUDRATE           UART_BAUDRATE_BAUDRATE_Baud1M
 +#define SER_PHY_UART_PARITY             false
-+#define SER_PHY_UART_BAUDRATE           UART_BAUDRATE_BAUDRATE_Baud115200
+ #define SER_PHY_UART_BAUDRATE           UART_BAUDRATE_BAUDRATE_Baud1M
```

Comment:
Just curious: A baud rate of 1m is defined here, but we end up with two hex files (both 115k2 and 1m). Do you manually edit this line when creating the hex files?

Reply:
Yes, we have a step on the build server that search-and-replaces the baud rate, and then recompiles